add-on/preferences/prefs.js

```diff
@@ -1,7 +1,12 @@
 pref("loop.enabled", true);
 pref("loop.remote.autostart", true);
+#ifdef LOOP_DEV_XPI
```

Comment:
Might be simpler to just use DEV_XPI here too, to keep the define names consistent across various places.  Up to you.